commit 129588442041c32b0d98f2361a69e0e350592f70
Author: Oskar Kruschitz <okr@huemer-it.com>
Date:   Thu Dec 13 16:13:01 2018 +0100

    Added missing Include
    
    Fixing an error during Build with Qt5.12:
    error: member access into incomplete type 'const QWebEngineCertificateError'

diff --git src/gui/wizard/webview.cpp src/gui/wizard/webview.cpp
index 15c1f314e..73e671b8f 100644
--- src/gui/wizard/webview.cpp
+++ src/gui/wizard/webview.cpp
@@ -10,6 +10,7 @@
 #include <QProgressBar>
 #include <QLoggingCategory>
 #include <QLocale>
+#include <QWebEngineCertificateError>
 
 #include "common/utility.h"
 
